src/bare-metal/microcontrollers/mmio.md

```diff
@@ -0,0 +1,5 @@
+# Raw MMIO
```

Comment:
Aha, MMIO is short for "memory-mapped I/O" according to https://en.wikipedia.org/wiki/Memory-mapped_I/O_and_port-mapped_I/O. I wanted to suggest adding a speaker note about it, but I cannot add it with a comment since the last like ends with triple-backtick and that breaks the formatting of the suggestion :slightly_smiling_face:

Reply:
Added a note at the top of the page.